modules/directives/src/main/scala/scala/build/preprocessing/directives/ComputeVersion.scala

```diff
@@ -0,0 +1,45 @@
+package scala.build.preprocessing.directives
+
+import scala.build.EitherCps.{either, value}
+import scala.build.Ops.EitherOptOps
+import scala.build.Positioned
+import scala.build.directives.{
+  DirectiveDescription,
+  DirectiveExamples,
+  DirectiveGroupName,
+  DirectiveLevel,
+  DirectiveUsage,
+  HasBuildOptions
+}
+import scala.build.errors.BuildException
+import scala.build.options.{BuildOptions, ComputeVersion => cv, SourceGeneratorOptions}
+import scala.cli.commands.SpecificationLevel
+
+@DirectiveGroupName("Compute Version")
+@DirectiveExamples("//> using computeVersion git:tag")
+@DirectiveExamples("//> using computeVersion command:'echo 1.0.0'")
+@DirectiveUsage("//> using computeVersion git:tag", "`//> using platform `git:tag")
```

Comment:
What is ` using platform `git:tag`  used for?  Could we actually separate computeVersion into a separate PR?

Reply:
`platform` is a mistake here. I could extract the commits with computeVersion to a different PR, I'll do that